### DISCUSSION TITLE
Suggetion for removal of diacritics on some languages

### DISCUSSION THREAD

**Discussion by Synneks** [monkeytypegame/monkeytype#1017] [Ideas] [answered]
Hello,

I appreciate your work and I would like to make a suggestion. 
I would want to type in romanian and german but without the diacritics, since we never really use them when texting in those languages.

**Comment by Miodec** [comment, marked as answer]
You mean completely removing all the special characters from the languages that make them what they are? I'm not exactly sure if everyone would like this idea.

**Reply by frogfrogfrogfrogfrog** [reply]
Maybe an option to turn on and off diacritics?

**Reply by SeerLite** [reply]
I would like this one too, these symbols aren't always used when texting (Spanish here). I think stats should be stored separately (like numbers and punctuation). Not everyone is looking to only improve their "texting" speed, but their formal typing speed too, and it'd be unfair to compare the two.

This could apply to English too now that I think about it. Some people don't really care about apostrophes.

**Reply by Synneks** [reply]
A toggle or maybe separate languages like German and German w/o diacritics

**Reply by molohov** [reply]
A toggle is an excellent idea. This applies to modern Pinyin also, see #1719

**Reply by Miodec** [reply]
Added #1755
